src/Microsoft.VisualBasic/tests/OperatorsTests.cs

```diff
@@ -5456,11 +5456,25 @@ private static void AssertEqual(object expected, object actual)
         {
             if (expected is double expectedDouble && actual is double actualDouble)
             {
-                Assert.Equal(expected.ToString(), actual.ToString());
+                if (expectedDouble == 0)
+                {
+                    Assert.Equal(0, actualDouble);
+                }
+                else
+                {
+                    Assert.Equal(expected.ToString(), actual.ToString());
```

Comment:
As from before, this shouldn't be doing a string comparison, as that will end up truncating bits and doing a relative comparison.

Reply:
Comment on an outdated commit, just look at the second commit now